.github/workflows/build.yml

```diff
@@ -36,25 +36,24 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-#        os: [windows-latest, macos-latest, ubuntu-latest]
-        os: [ubuntu-latest]
+        #        os: [windows-latest, macos-latest, ubuntu-latest]
+        os: [windows-latest, ubuntu-latest]
     runs-on: ${{ matrix.os }}
     steps:
       - name: Prepare git
         run: git config --global core.autocrlf false
         if: startsWith(matrix.os, 'windows')
 
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Set up JDK 17
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
           distribution: temurin
           java-version: 17
           cache: 'maven'
 
       - name: Build with Maven
-        run: mvn -B clean verify -Dno-format
+        run: mvn -B clean install -Dno-format
 
-# Native won't work anyway
-#      - name: Build with Maven (Native)
-#        run: mvn -B verify -Dnative -Dquarkus.native.container-build -Dnative.surefire.skip -Dno-format
+        #- name: Build with Maven (Native)
+        #run: mvn -B install -Dnative -Dquarkus.native.container-build -Dnative.surefire.skip
```

Comment:
Perhaps leave it uncommented so you can track when the native build starts working?

Reply:
I was going to leave this for now because I think its really far from Native working yet.  I want to do some testing locally first.